src/test/java/nextstep/subway/SubwayApi.java

```diff
@@ -0,0 +1,94 @@
+package nextstep.subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.dto.line.LineRequest;
+import nextstep.subway.dto.line.LineResponse;
+import nextstep.subway.dto.station.StationRequest;
+import org.springframework.http.MediaType;
+
+import java.util.List;
+
+public class SubwayApi {
+
+    public static ExtractableResponse<Response> 지하철역_생성(String stationName) {
+        StationRequest stationRequest = new StationRequest(stationName);
+
+        return RestAssured.given().log().all()
+                .body(stationRequest)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/stations")
+                .then().log().all()
+                .extract();
+    }
+
+    public static ExtractableResponse<Response> 지하철역_삭제(Long stationId) {
+        return RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().delete("/stations/" + stationId)
```

Comment:
```suggestion
                .when().delete("/stations/{stationId}", stationId)
```

와 같이 작성한다면 조금 더 친절한 로그를 보실 수 있습니다~

Reply:
이건 몰랐네요!! 꿀팁 감사합니다!